src/lib/FieldTypeProcessorInterface.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace EzSystems\EzPlatformRest;
+
+interface FieldTypeProcessorInterface
```

Comment:
In the same context you have `\EzSystems\EzPlatformRest\RequestParser` which creates naming mess. An interface knows it's an interface, Hungarian-ish notation is not needed.
One of the new ways of overcoming redundancy in names is a separate contracts namespace documented [here](https://doc.ibexa.co/en/latest/community_resources/package_structure/#contracts).